build_tools/github_actions/test_executable_scripts/test_hipfft.py

```diff
@@ -21,16 +21,18 @@
 # If smoke tests are enabled, we run smoke tests only.
 # Otherwise, we run the normal test suite
 test_type = os.getenv("TEST_TYPE", "full")
-if test_type == "smoke":
-    test_filter = ["--smoketest"]
-else:
-    # "--test_prob" is the probability that a given test will run.
-    # Due to the large number of tests for hipFFT, we only run a subset.
-    test_filter = [
-        "--gtest_filter=-*multi_gpu*",
-        "--test_prob",
-        "0.02",
-    ]
+test_filter = []
+if test_type != "full":
+    if test_type == "smoke":
+        test_filter = ["--smoketest"]
+    else:
+        # "--test_prob" is the probability that a given test will run.
+        # Due to the large number of tests for hipFFT, we only run a subset.
+        test_filter = [
+            "--gtest_filter=-*multi_gpu*",
+            "--test_prob",
+            "0.02",
+        ]
```

Comment:
could we do:

```suggestion
if test_type == "full":
     test_filter = []
elif test_type == "smoke":
     test_filter = ["--smoketest"]
else:
     # "--test_prob" is the probability that a given test will run.
        # Due to the large number of tests for hipFFT, we only run a subset.
        test_filter = [
            "--gtest_filter=-*multi_gpu*",
            "--test_prob",
            "0.02",
        ]
```
?

for all the following ones too, personally easier for me to read